src/mono/wasm/Wasm.Build.Tests/WasmTemplateTests.cs

```diff
@@ -46,6 +46,10 @@ private void UpdateBrowserMainJs(string targetFramework)
                     targetFramework == "net8.0"
                         ? ".withConsoleForwarding().withElementOnExit().withExitCodeLogging().withExitOnUnhandledError().create()"
                         : ".withConsoleForwarding().withElementOnExit().withExitCodeLogging().create()");
+
+            if (flatOutput)
```

Comment:
What case is this useful for?

Reply:
The location for `dotnet.js` import changes when user sets the `WasmRuntimeAssetsLocation`.

Reply:
Instead of `flatOutput` pass in a string with the relative location for runtime assets, and the default can be `./_framework`. This way the test passing a different value essentially is testing setting a non-default value, but that is explicit in the test.